src/cswinrt/main.cpp

```diff
@@ -35,6 +35,7 @@ namespace cswinrt
         { "target", 0, 1, "<net6.0|net5.0|netstandard2.0>", "Target TFM for projection. Omit for compatibility with newest TFM (net5.0)." },
         { "component", 0, 0, {}, "Generate component projection." },
         { "verbose", 0, 0, {}, "Show detailed progress information" },
+        { "internal", 0, 0, {}, "Generate the projection as internal."},
```

Comment:
private rather than internal for it to flow with our other terminology?  And maybe want to differentiate in the description from the one for embedded?

Reply:
I tried private at first, it's a reserved keyword

Reply:
Do you have an idea for the description? They do the same thing from cswinrt.exe 's POV so not sure

Reply:
oh thats interesting.

Reply:
Embedded generates the projection as internal when trying to embed the projection and runtime. 

Internal generates the projection as internal when just making the projection private.

?

Reply:
Generates a private projection and Generates an embedded projection and then in docs we can define how they defer?

Reply:
or since we use internal rather than private, we can leave it is as for the internal one.  And update the embeded projection text.